netlify/edge-functions/star-search.tsx

```diff
@@ -1,13 +1,27 @@
 import React from "react";
 import { ImageResponse } from "og_edge";
+import { parse, pipe, string, uuid } from "valibot";
 import type { Config } from "https://edge.netlify.com";
 import { getLocalAsset } from "../og-image-utils.ts";
 
 const MAX_CHARS = 130;
+const baseApiUrl = Deno.env.get("NEXT_PUBLIC_API_URL");
+
+const UuidSchema = pipe(string(), uuid());
```

Comment:
This is redefined here because although edge functions are served from the same domain, they are distinct apps/functions. The edge functions have no knowledge of the Next.js codebase and therefore cannot import code from it.